sentences/de/_common.yaml

```diff
@@ -37,7 +37,7 @@ lists:
     range:
       type: "temperature"
       from: 0
-      to: 100
+      to: 300
```

Comment:
If we are thinking about smart oven, we should also consider smart fridges and decrease the lower boundary, too ^^

Reply:
Thaha, thanks for that :D For now i changed it to -30, that should be enough i guess.